### PR TITLE
Split balance screen into two steps

### PR DESCRIPTION
**Pull request by thomaseizinger**
This PR splits the current balance screen into two steps. The first page now only includes the balances of each user.
Upon navigating to one of the users (by clicking somewhere on the ListItem), a detail view is opened.

|Overview|Tina|Thomas E.|Thomas B.|
|----|----|----|----|
|localhost-3000-3xtc8mkngsqp-balances-tina iphone 5 1|localhost-3000-3xtc8mkngsqp-balances-tina iphone 5 2|localhost-3000-3xtc8mkngsqp-balances-tina iphone 5 3|localhost-3000-3xtc8mkngsqp-balances-tina iphone 5 4|

The PR also changes the icons on the main screen:
localhost-3000-3xtc8mkngsqp-balances-tina iphone 5

We now use the group icon and the icon for account balances. The text of the button also changed.

Besides these features, the PR also re-enables **hot reloading**, at least for styling changes. Changes to the JS are not yet reflected. A page-reload is still needed for those changes. However, I gained some knowledge concerning the HMR process. Let's see if I can get JS hot-reloading to work again too :D

Note: I discovered a minor styling issue that is introduced by this PR for the __existing__ codebase. However, this should be resolved, as soon as the other PRs are merged. But just to be sure, we should check back once those are merged.

### CODE REVIEW

File: src/routes/App/modules/currentNobt/selectors.js [hunk 1]
```diff
@@ -73,15 +72,42 @@ export const getFilteredBills = createSelector([ getBills, getBillFilter, getBil
...
 export const makeGetBill = () => createSelector( [getBills, getBillId], (bills, billId) => {
   return bills
-    .filter( bill => bill.id == billId )
+    .filter( bill => bill.id === billId )
     .map(deNormalizeBill)
-    .find(() => true); // equal to .first() :)
+    .find(first);
 });
 
+const getBalanceOwner = (state, props) => props.params.name;
```
Comment (thomaseizinger) [inline src/routes/App/modules/currentNobt/selectors.js]: We should access the path-parameter from the route definition here.

File: src/routes/App/modules/currentNobt/selectors.js [hunk 2]
```diff
@@ -73,15 +72,42 @@ export const getFilteredBills = createSelector([ getBills, getBillFilter, getBil
 
 });
 
-const getBillId = (state, props) => props.params.billId;
+let first = () => true;
+
+const getBillId = (state, props) => parseInt(props.params.billId);
```
Comment (thomaseizinger) [inline src/routes/App/modules/currentNobt/selectors.js]: Same here.

File: src/styles/_resets.scss [hunk 3]
```diff
@@ -0,0 +1,7 @@
+@import "~react-toolbox/lib/globals";
+
+h1, h2, h3, h4, h5 {
```
Comment (thomaseizinger) [inline src/styles/_resets.scss]: I am not sure about this. What do you think @duffleit?
Should we override these styles in general, or make these styling changes component styles of the `Title` and `SubTitle` component?

**Comment by duffleit** [comment]
I know we all hate this discussion. However, we should think of removing this minus if someone owes money. The red Fond Colour should be enough imho. Because owing a negative amount - could literally be interpreted as getting money.

**Comment by thomaseizinger** [comment]
Depends. There are no transaction on the balance screen, just balances. The minus is essential there.
On the more detailed screen, I would rather prefer to find a different word for "transaction" and rather suggest the user that these numbers are the balance between yourself and the mentioned person. So for example, my balance with Tina is positive because she owes me money. Tina's balance with me on the other hand is negative, because she owes me money.

File: src/routes/App/modules/currentNobt/selectors.js [hunk 4]
```diff
@@ -74,15 +75,42 @@ export const getFilteredBills = createSelector([ getBills, getBillFilter, getBil
...
+const getBalanceOwner = (state, props) => props.params[balanceDetailPathVariable];
+
+export const makeGetBalance = () => createSelector( [getBalances, getBalanceOwner], (balances, balanceOwner) => {
+  return balances
+    .filter( balance => balance.me.name === balanceOwner )
+    .find(first)
+} );
+
+export const getSumOfPaidBills = (paidBills) => {
+  console.log(paidBills)
```
Comment (duffleit) [inline src/routes/App/modules/currentNobt/selectors.js]: remove

Comment (thomaseizinger) [inline src/routes/App/modules/currentNobt/selectors.js]: Done

**Pull request merged**